### ISSUE TITLE
Audit platform to ensure it adheres to accessibility standards and best practices

### ISSUE THREAD

**Issue by AoifeHughes** [alan-turing-institute/AssurancePlatform#179]

**Comment by AoifeHughes** [comment]
Overlaps with https://github.com/alan-turing-institute/AssurancePlatform/issues/170

**Comment by chrisdburr** [comment]
I'm hoping this will be something for the UI/UX agency to support (e.g. ensuring their design recommendations are accessible). Unlikely to be much progress here until this work begins.

**Comment by kallewesterling** [comment]
This issue is too vague. First, it seems to be overlapping with two issues #180 and #170 so we might want to think about whether this issue (#179) is actually included in the work we'll do in #180 (after splitting it, especially).

#170 looks like it might be part of an outcome of #180, which is further confusing. I'll add a comment in #180 about that.

**Comment by chrisdburr** [comment]
I see this issue as being more about accessibility in the sense used in guidance such as the WCAG 2 standard (https://www.w3.org/WAI/standards-guidelines/wcag/), rather than the more specific notion of accessibility used in FAIR (i.e. accessible data). So, I think we can keep this and #180 as separate. 

However, #170 is related, because we should try to ensure that any new components we build into the site are accessible by design.

**Comment by kallewesterling** [comment]
I forgot to add here, in my previous comment, that it needs to have a different title: it should be an active verb and something that is more precise than "accessibility best practices", i.e. "audit platform to ensure that it follows the WCAG 2 standard".

**Comment by kallewesterling** [comment]
This is a prerequisite that we'll keep as part of the acceptance criteria for our user stories. (I am creating templates for GitHub issues.)
